modular_skyrat/modules/customization/modules/jobs/_job.dm

```diff
@@ -75,6 +92,7 @@
 
 /datum/job/head_of_security
 	banned_quirks = list(SEC_RESTRICTED_QUIRKS, HEAD_RESTRICTED_QUIRKS)
+	banned_augments = list(SEC_RESTRICTED_AUGMENTS)
```

Comment:
I'd like for these to be applied to command as well, mainly because if someone wants to be in command, I'd like to think they'd be provided at the very least a shitty prosthetic to make them at least somewhat capable of using all their limbs.

Reply:
Sure